routes/account/account.js

```diff
@@ -0,0 +1,97 @@
+const express = require('express');
+
+const router = express.Router();
+const {
+  createAccount,
+  updateAccount,
+  deleteAccount,
+  getAccounts
+} = require('../../controllers/accountControllers/userAccountController');
+
+const { adminGuard } = require('./../../middlewares/adminAuth');
+const auth = require('./../../middlewares/passportAuth');
+
+router.get('/', auth, adminGuard ,getAccounts);
+router.post('/', auth, createAccount);
+router.put('/:id', auth, updateAccount);
+router.delete('/:id', auth, deleteAccount);
+
+module.exports = router;
+
+
+
+
+
+
+
+
+
+
+
+
+
+// const express = require('express');
```

Comment:
Please remove unused code

Reply:
Sorry for my ignorance, I removed it and consider that next time